### PR TITLE
DISTX-719:Add cloudbreak env host port for datalake template

### PR DESCRIPTION
**Pull request by satyabollineni**
This change is required for the thunderhead change : https://github.infra.cloudera.com/thunderhead/thunderhead/pull/6978/files

### CODE REVIEW

**Review by lajosrodek — APPROVED** [review]

**Review by lajosrodek — CHANGES_REQUESTED** [review]
The docker-compose change looks alright, but this will not work unless the thunderhead docker image tag `1.0.0-b5510` gets updated with the appropriate value. In other words, this PR depends on https://github.infra.cloudera.com/thunderhead/thunderhead/pull/6978 having been merged. Once that change is in and the new thunderhead service images have been published:

1. The image tag shall be replaced in [cloudbreak.bash](https://github.com/hortonworks/cloudbreak-deployer/blob/master/include/cloudbreak.bash#L74-L80)
2. cbd recompiled locally & tested that `datalake-api` picks up the service dependency correctly
3. If tests are successful, this PR can be updated with the change of Step 1

**Comment by satyabollineni** [comment]
> The docker-compose change looks alright, but this will not work unless the thunderhead docker image tag `1.0.0-b5510` gets updated with the appropriate value. In other words, this PR depends on https://github.infra.cloudera.com/thunderhead/thunderhead/pull/6978 having been merged. Once that change is in and the new thunderhead service images have been published:
> 
> 1. The image tag shall be replaced in [cloudbreak.bash](https://github.com/hortonworks/cloudbreak-deployer/blob/master/include/cloudbreak.bash#L74-L80)
> 2. cbd recompiled locally & tested that `datalake-api` picks up the service dependency correctly
> 3. If tests are successful, this PR can be updated with the change of Step 1

Thanks for the inputs here. Since the changes are promoted to mow-dev,int,stage and prod envs i.e. adding the cloudbreak host and port to datalake api , I am going ahead and merging the thunderhead PR now https://github.infra.cloudera.com/thunderhead/thunderhead/pull/6978 and will then update the image in cloudbreak.bash as mentioned here.

File: include/cloudbreak.bash [hunk 1]
```diff
@@ -71,13 +71,13 @@ cloudbreak-conf-tags() {
...
-    env-import DOCKER_TAG_DISTROX_API 1.0.0-b6009
-    env-import DOCKER_TAG_AUDIT 1.0.0-b6009
-    env-import DOCKER_TAG_DATALAKE_DR 1.0.0-b6009
+    env-import DOCKER_TAG_IDBMMS 1.0.0-b6156
+    env-import DOCKER_TAG_WORKLOADIAM 1.0.0-b6156
+    env-import DOCKER_TAG_ENVIRONMENTS2_API 1.0.0-b6156
+    env-import DOCKER_TAG_DATALAKE_API 1.0.0-b6156
+    env-import DOCKER_TAG_DISTROX_API 1.0.0-b6156
+    env-import DOCKER_TAG_AUDIT 1.0.0-b6156
+    env-import DOCKER_TAG_DATALAKE_DR 1.0.0-b6156
```
Comment (lajosrodek) [inline include/cloudbreak.bash:80]: [1.0.0-b6156](https://release.infra.cloudera.com/manifest_info?stack=THUNDERHEAD&release=1.0.0-b6156) is a failed build on the RE Portal. I think [1.0.0-b6159](https://release.infra.cloudera.com/manifest_info?stack=THUNDERHEAD&release=1.0.0-b6159) is the earliest green one containing your change.

**Review by lajosrodek — CHANGES_REQUESTED** [review]

File: include/cloudbreak.bash (hunk 1 above)
Comment (satyabollineni) [reply include/cloudbreak.bash]: Updated

**Review by lajosrodek — APPROVED** [review]
Thanks, LGTM

**Pull request merged**